test/iron-menu-behavior.html

```diff
@@ -100,6 +113,53 @@
           }, 200);
         });
 
+        test('keyboard events should not bubble', function(done) {
+          var menu = fixture('nested');
+          var keyCounter = 0;
+          var firstNestedMenu = menu.firstElementChild;
+
+          function triggerKeyEvent(keyCode, shift) {
```

Comment:
oh. it seems like this event now bubbles in MockInteractions: https://github.com/PolymerElements/iron-test-helpers/blob/master/mock-interactions.js#L147. 

 I updated the test. Also, I found that a few of the old tests are failing in Firefox. I can't reproduce the issue locally, but it seems like a timing issue.  This issue is present in #master as well.